### PR TITLE
Suppress splash screen when Playnite starts at Windows startup

### PR DESCRIPTION
**Pull request by craigt00**
For #1285 

Added a new startup argument named 'hidesplashscreen' which suppresses the splash screen when Playnite starts.  This parameter is added to the Playnite.lnk file which is generated when a user selects to start Playnite at windows startup (via Options...), thereby preventing the splash screen from appearing at Windows startup but retains the splash screen if Playnite is started manually.

### CODE REVIEW

File: source/Playnite/Settings/PlayniteSettings.cs [hunk 1]
```diff
@@ -1351,7 +1351,7 @@ public static void SetBootupStateRegistration(bool runOnBootup)
             if (runOnBootup)
             {
                 FileSystem.DeleteFile(shortcutPath);
-                Programs.CreateShortcut(PlaynitePaths.DesktopExecutablePath, "", "", shortcutPath);
+                Programs.CreateShortcut(PlaynitePaths.DesktopExecutablePath, "--hidesplashscreen", "", shortcutPath);
```
Comment (JosefNemec) [inline source/Playnite/Settings/PlayniteSettings.cs]: Create new `CmdLineOptions` object with appropriate settings and then pass the result via `ToString` method instead for direct string.

Comment (craigt00) [reply source/Playnite/Settings/PlayniteSettings.cs]: No worries.  Pull request updated.

**Comment by kremuwakarz** [comment]
I've replaced explorer.exe for Playnite.FullscreenApp.exe in windows shell so Playnite can now launch much faster! this is better than "run playnite when computer starts" option however the splash screen is annoying so I need a real ini setting for disabling splash screen or at least paramater for shortcut like -hidesplashscreen (so I could use .bat file to run Playnite.

**Comment by JosefNemec** [comment]
https://github.com/JosefNemec/Playnite/wiki/Cmdline-arguments

**Pull request merged**